hs_tracking/middleware.py

```diff
@@ -0,0 +1,11 @@
+from .models import Session, Variable
+
+class Tracking(object):
+    """If a page implements "can_view(request)" we honor the permission and
+    raise a 403 if the logged in user would normally not be able to view the
+    content"""
```

Comment:
@ironfroggy I am a bit confused on the comment here which does not seem to be handled currently in this Tracking middleware. process_response() method just records visit request path. Is there more planned to be done here in this middleware?